pkg/policy/l4.go

```diff
@@ -33,6 +33,25 @@ var (
 // L7DataMap contains a map of L7 rules per endpoint where key is a hash of EndpointSelector
 type L7DataMap map[api.EndpointSelector]api.L7Rules
 
+func (l7 L7DataMap) MarshalJSON() ([]byte, error) {
```

Comment:
Don't we need the Unmarshall?

Reply:
@aanm @joestringer @nebril Saw this only after the merge, please file a github issue if required.

Reply:
@aanm Do we ever directly import L4Policy as JSON? I figured that has to go through the API.